src/index.js

```diff
@@ -17,6 +17,12 @@ const invokeCallback = ( events, name, params ) => {
 	}
 }
 
+const getPreviewHtml = ( title, lang, callback ) => {
+	requestPagePreview( lang, title, isTouch, data => {
```

Comment:
Is it worth adding some basic truthy input validation here for `title` and `lang` before executing `requestPagePreview`?

Reply:
I feel this kind of situation should be handled upstream but not calling this function unless all inputs are available.